core/types/eth_transaction.go

```diff
@@ -40,9 +41,10 @@ import (
 type EthTransaction struct {
 	data ethTxdata
 	// caches
-	hash atomic.Value
-	size atomic.Value
-	from atomic.Value
+	hash  atomic.Value
```

Comment:
Actually in harmony, EthTransaction is exactly the ethereum transaction format. So the code here should be very similar to the reference implementation in /core/types/transaction.go

Reply:
yeah, and so it is. I tried to migrate&merge all relevant changes from upstream core/types/transaction.go. If you think there is some code missing here, please elaborate.